src/app/components/MostRead/Canonical/LastUpdated/index.tsx

```diff
@@ -30,6 +30,7 @@ const LastUpdated = ({
     locale={locale}
     timezone={timezone}
     padding={false}
+    altCalendar={null}
```

Comment:
This is a mismatch between using JSX components in TSX. `altCalendar` needs to be set to basically an `any` type in the JSX component (by not giving it a default value) so that TSX components can pass in either a `null` or altCalendar function.

Its not great, but its just a weirdness of compatibility between JSX and TSX types.